### PR TITLE
Enable test/ecall_ocall on windows

### PR DESCRIPTION
**Pull request by mingweishih**
This PR fixes minor building issues and enables the test/ecall_ocall windows. Seems the test is no longer blocked by the build failure on clangw as described in #39.

### CODE REVIEW

**Comment by BRMcLaren** [comment]
bors try

**Comment by BRMcLaren** [comment]
Hey @mingweishih ,

I wanted to check if you were aware that we have a bot for testing on all supported platforms? It might be beneficial to run before reviewing.

**Comment by BRMcLaren** [comment]
Clicked Close instead of "comment", reopening.

File: tests/ecall_ocall/host/host.cpp [hunk 1]
```diff
@@ -250,7 +250,7 @@ static void test_cross_enclave_calls()
         enc_cross_enclave_call(enclave_wrap::get(0), &total, 0, SEED, 0);
     OE_TEST(OE_OK == result);
 
-    printf("total=%u, expected_total=%lu\n", total, expected_total);
+    printf("total=%u, expected_total=%zu\n", total, expected_total);
```
Comment (johnkord) [inline tests/ecall_ocall/host/host.cpp]: You know I've been programming with C for a long time and I've never known about `z` in a format specifier. Very cool! I need to use this more often

Comment (mingweishih) [reply tests/ecall_ocall/host/host.cpp]: Thanks John! I think it's specifically designed for size_t :)

**Pull request merged**